packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -91,6 +101,23 @@ const CFDsListing = observer(() => {
     const { has_svg_accounts_to_migrate } = useMT5SVGEligibleToMigrate();
     const getAuthStatus = (status_list: boolean[]) => status_list.some(status => status);
 
+    const { data: tradingPlatformStatusData } = useTradingPlatformStatus();
+
+    const getTradingPlatformStatus = (platform?: string) => {
+        const status = tradingPlatformStatusData?.find(p => p.platform === platform)?.status;
+
+        switch (status) {
+            case TRADING_PLATFORM_STATUS.MAINTENANCE:
+                return setServerMaintenanceModal(true);
+            case TRADING_PLATFORM_STATUS.UNAVAILABLE:
+                return setAccountUnavailableModal(true);
+            case TRADING_PLATFORM_STATUS.ACTIVE:
+                return getAccount();
+            default:
+                break;
+        }
+    };
```

Comment:
Don't you need a `break` at every case statement? I assume if the statement is true then you want the function to stop executing.